### ISSUE TITLE
The convergence of SRCNN

### ISSUE THREAD

**Issue by niuqun** [togheppi/pytorch-super-resolution-model-collection#2]
Dear all,

I just run the SRCNN with designated dataset using my own computer. I use default settings in my experiment. However, it seems that the loss remains stable from the beginning. Any advice with the training of SRCNN?

**Comment by micklexqg** [comment]
do you have run the SRCNN code successfully?
I think There are a lot of errors to be corrected especially for loading data.
